colossus-docs/src/main/scala/StreamingHttpExample.scala

```diff
@@ -1,19 +1,22 @@
+import akka.actor.ActorSystem
 import colossus._
-
-import protocols.http._
-import protocols.http.streaming._
-import core._
-import service._
+import colossus.core._
+import colossus.protocols.http._
+import colossus.protocols.http.filters.HttpStreamCustomFilters
+import colossus.protocols.http.streaming._
+import colossus.service._
 import colossus.streaming._
 
-object StreamServiceExample {
+object StreamServiceExample extends App {

```

Comment:
I suppose this is normally run from `Main`.

Reply:
Other examples are runnable too like HttpServiceExample, HelloWorld, Filter example. I think examples that can run in a standalone manner are helpful